tests/drivers/build_all/ieee802154/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_TEST=y
+CONFIG_TEST_USERSPACE=y
+CONFIG_LOG=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_GPIO=y
+CONFIG_SPI=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_IEEE802154=y
+CONFIG_NET_L2_IEEE802154_SECURITY=y
+
+CONFIG_IEEE802154_CC1200=y
+CONFIG_IEEE802154_CC2520=y
```

Comment:
Please enable build test for all 802.15.4 drivers (CC13xx, DW1000, KW41z, etc.)

Reply:
Done, requires adding many test configs though, as half of the drivers are for SoC peripherals, not external modems.